variables.tf

```diff
@@ -163,6 +163,11 @@ variable "workers_additional_policies" {
   default     = []
 }
 
+variable "worker_enable_instance_refresh" {
```

Comment:
Rather than have this module-wide, could we make this a per-ASG setting?

Reply:
Good point. Agreed.

Reply:
Sounds good.

Reply:
Dropped the global variable. One can set `instance_refresh_enabled` globally with:

``` 
workers_group_defaults = {
    "instance_refresh_enabled" : true
  }
```